SwiftLint/swiftlint.yml

```diff
@@ -152,6 +152,11 @@ custom_rules:
       - comment
       - doccomment
       - doccomment.field
+  unnecessary_empty_type_in_case:
+    name: "Unnecessary empty type in case"
+    regex: '\(_\):'
+    message: "Unnecessary empty type in case. Remove '(_)' from case"
+    severity: warning
```

Comment:
I'm don't think that we should enforce it as a swiftlint rule. I even like to leave the (_) to show that there _is_ an associated value but we don't need it.

@alex-taras any opinion ?